clients/client-accessanalyzer/protocols/Aws_restJson1_1.ts

```diff
@@ -2865,10 +2865,13 @@ const serializeAws_restJson1_1FilterCriteriaMap = (
   input: { [key: string]: Criterion },
   context: __SerdeContext
 ): any => {
-  return Object.keys(input).reduce((acc: any, key: string) => {
-    acc[key] = serializeAws_restJson1_1Criterion(input[key], context);
-    return acc;
-  }, {});
+  return Object.entries(input).reduce(
+    (acc: { [key: string]: Criterion }, [key, value]: [string, any]) => ({
+      ...acc,
+      [key]: serializeAws_restJson1_1Criterion(value, context)
+    }),
```

Comment:
Gonna leave another comment for the newlines here. Otherwise LGTM

Reply:
As discussed offline, the newlines are introduced by prettier.
We're discussing increasing prettier [printWidth](https://prettier.io/docs/en/options.html#print-width) to 120, but it would be taken up in a different PR